Source/WebCore/PAL/pal/spi/cocoa/PassKitSPI.h

```diff
@@ -71,6 +71,14 @@
 #endif
 #endif
 
+#if HAVE(PASSKIT_PAYMENT_ORDER_DETAILS)
+#if HAVE(PASSKIT_MODULARIZATION) && USE(APPLE_INTERNAL_SDK)
+#import <PassKitCore/PKPaymentRequestStatus.h>
+#else
+#import <PassKit/PKPaymentRequestStatus.h>
```

Comment:
this should already be handled by `#import <PassKit/PassKit.h>`.  is that not working?

also, the new `PKPaymentOrderDetails` is public API, so not really sure why we'd need to do anything special for it (i.e. this file is for handling SPI that public SDKs dont have, not API that they do)

Reply:
The immediate challenge is that the `<PassKit/PassKit.h>` import happens inside of an `#if PLATFORM(IOS_FAMILY)` block. I tried moving it out, and it revealed other duplicate declarations that would need cleanup, so went with this approach to keep the scope of this PR small.

Happy to try again, as it does feel like the right thing to do.

Reply:
Tried again, and was able to clean up a lot more, by relying on the umbrella `PassKit.h` import more and removing declarations we only needed for pre-Big Sur OSes. Let me know how that looks!